tests/cypress/page-object/exportDialog.ts

```diff
@@ -4,10 +4,10 @@ export class ExportDialog extends BaseComponent {
     static defaultSelector = '[data-cm-role="export-options"]';
 
     export(downloadsFolder:string, workspace:string = 'Staging content only', exportFormat: string = 'zip') {
-        const dialog = getComponentByAttr(BaseComponent, 'data-cm-role', 'export-options');
+        getComponentByAttr(BaseComponent, 'data-cm-role', 'export-options');
```

Comment:
ok to remove this full line as it's not used anymore.

Reply:
done